functions.php

```diff
@@ -43,6 +43,20 @@ function melany_setup() {
 	add_theme_support( 'automatic-feed-links' );
 
 	/**
+	 * Support title tag.
+	 *
+	 * @since future-release
+	 */
+	if ( ! function_exists( '_wp_render_title_tag' ) ) {
+		function theme_slug_render_title() {
```

Comment:
@wraxen you should replace `theme_slug_render_title` with `melany_render_title`.